Rubberduck.Parsing/Annotations/AnnotationType.cs

```diff
@@ -43,7 +64,7 @@ public enum AnnotationType
         PredeclaredId = 1 << 16 | Attribute | ModuleAnnotation,
         [AttributeAnnotation("VB_Exposed", "True")]
         Exposed = 1 << 17 | Attribute | ModuleAnnotation,
-        Obsolete = 1 << 18 | MemberAnnotation
+        Obsolete = 1 << 18 | MemberAnnotation | VariableAnnotation
```

Comment:
Based on `VariableAnnotationDeclarationTypes` variables and constants can be marked as obsolete. Does it make sense?

Reply:
I am talking about variables and constants appearing inside members.

Reply:
The declarations do not distinguish between variables and constants inside members and on module level. Since `Obsolete` makes sense for the later in my opinion I assembly the flag to the annotation type.